client/src/models/Location.js

```diff
@@ -53,6 +55,33 @@ export default class Location extends Model {
           return true
         })
         .default(null),
+      // Used for form validation only. Shouldn't be used elsewhere.
+      displayedCoordinate: yup
```

Comment:
It was named `mgrs` but vassil requested it to be more generic so that when we add new formats in future we will be able to use same field. I suggest keeping this way.

Reply:
Maybe customCoordinate, or otherCoordinate would be better than? @VassilIordanov what do you think? With displayed I would think that this is the only displayed coordinate.

Reply:
personally, I am fine with either